staging/src/k8s.io/apiserver/pkg/endpoints/handlers/get.go

```diff
@@ -103,24 +108,40 @@ func GetResource(r rest.Getter, e rest.Exporter, scope RequestScope) http.Handle
 			if trace != nil {
 				trace.Step("About to Get from storage")
 			}
-			return r.Get(ctx, name, &options)
+			obj, err = r.Get(ctx, name, &options)
+			return
 		})
 }
 
 // GetResourceWithOptions returns a function that handles retrieving a single resource from a rest.Storage object.
 func GetResourceWithOptions(r rest.GetterWithOptions, scope RequestScope, isSubresource bool) http.HandlerFunc {
 	return getResourceHandler(scope,
-		func(ctx request.Context, name string, req *http.Request, trace *utiltrace.Trace) (runtime.Object, error) {
+		func(ctx request.Context, name string, req *http.Request, trace *utiltrace.Trace) (obj runtime.Object, exported bool, err error) {
+			if values := req.URL.Query(); len(values) > 0 {
```

Comment:
Need I move the logic of extracting `exported` from `req.url.Query()` to another local private function? The following code block seems redundant with L88~102. OTOH, putting this into one local function doesn't benefits much except for reducing the  size of this file. WDYT?